src/gigs-board/components/molecule/markdown-editor.jsx

```diff
@@ -0,0 +1,30 @@
+const MarkdownEditor = ({
+  content,
+  onChange,
+}) => {
+  return (
+      <Widget
+        src="devhub.efiz.near/widget/SimpleMDE"
+        props={{
+          data: { content },
+          onChange,
+          toolbar: [
+            "heading",
+            "bold",
+            "italic",
+            "quote",
+            "code",
+            "link",
+            "unordered-list",
+            "ordered-list",
+            "checklist",
+            "mention",
```

Comment:
mention used to have an auto-completion. Is it possible to keep that?
<img width="1220" alt="image" src="https://github.com/near/neardevhub-widgets/assets/13259400/037cd529-b01f-49f5-b15e-aa86cec89b6a">